pkg/collector/metric/container_metric.go

```diff
@@ -140,7 +140,17 @@ func (c *ContainerMetrics) ResetCurr() {
 // NOTICE: can lose main container info for multi-container pod
 func (c *ContainerMetrics) SetLatestProcess(cgroupPID, pid uint64, comm string) {
 	c.CGroupPID = cgroupPID
-	c.PIDS = append(c.PIDS, pid)
+
+	notexist := true
+	for _, v := range c.PIDS {
```

Comment:
This PID list was being used to calculate GPU power consumption and PR #345 removes this dependency.

So can you add a `//TODO: review if we can remove the PIDS list` or something like that?

Reply:
ok , updated